### PR TITLE
r-phytools: new package at 0.6-60

### PR DESCRIPTION
**Pull request by molecuul**
depends on packages introduced in #9764, #9765

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-phytools/package.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+from spack import *
+
+
+class RPhytools(RPackage):
+    """Phylogenetic Tools for Comparative Biology (and Other Things)"""
+
+    homepage = "http://github.com/liamrevell/phytools"
+    url      = "https://cran.r-project.org/src/contrib/phytools_0.6-60.tar.gz"
+
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/r-phytools/package.py]: like #9767 , please add list_url

**Comment by molecuul** [comment]
@scheibelp could you look at this this one and #9764?
They are dependents of #9767 which has already been merged

File: var/spack/repos/builtin/packages/r-phytools/package.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class RPhytools(RPackage):
+    """Phylogenetic Tools for Comparative Biology (and Other Things)"""
+
+    homepage = "http://github.com/liamrevell/phytools"
+    url      = "https://cran.r-project.org/src/contrib/phytools_0.6-60.tar.gz"
+    list_url = "https://cran.r-project.org/src/contrib/Archive/phytools/"
+
+    version('0.6-60', sha256='55cad759510d247ebbf03a53a46caddadd3bf87584ccf7fcd6dd06d44516b377')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-phytools/package.py]:
```suggestion

    depends_on('r@3.2.0:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/r-phytools/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    depends_on('r-animation', type=('build', 'run'))
+    depends_on('r-ape', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-phytools/package.py]:
```suggestion
    depends_on('r-ape@4.0:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/r-phytools/package.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    depends_on('r-clustergeneration', type=('build', 'run'))
+    depends_on('r-coda', type=('build', 'run'))
+    depends_on('r-combinat', type=('build', 'run'))
+    depends_on('r-expm', type=('build', 'run'))
+    depends_on('r-maps', type=('build', 'run'))
+    depends_on('r-mnormt', type=('build', 'run'))
+    depends_on('r-numderiv', type=('build', 'run'))
+    depends_on('r-phangorn', type=('build', 'run'))
+    depends_on('r-plotrix', type=('build', 'run'))
+    depends_on('r-scatterplot3d', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-phytools/package.py]: Missing a few dependencies:
```suggestion
    depends_on('r-scatterplot3d', type=('build', 'run'))
    depends_on('r-mass', type=('build', 'run'))
    depends_on('r-nlme', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/r-phytools/package.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    depends_on('r-animation', type=('build', 'run'))
+    depends_on('r-ape', type=('build', 'run'))
+    depends_on('r-clustergeneration', type=('build', 'run'))
+    depends_on('r-coda', type=('build', 'run'))
+    depends_on('r-combinat', type=('build', 'run'))
+    depends_on('r-expm', type=('build', 'run'))
+    depends_on('r-maps', type=('build', 'run'))
+    depends_on('r-mnormt', type=('build', 'run'))
+    depends_on('r-numderiv', type=('build', 'run'))
+    depends_on('r-phangorn', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-phytools/package.py]:
```suggestion
    depends_on('r-phangorn@2.3.1:', type=('build', 'run'))
```

**Pull request merged**